<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scopeRangeEnterDialog</class>
 <widget class="QDialog" name="scopeRangeEnterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Scope Range</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="voltageSpinBox" name="vMaxBox">
         <property name="minimum">
          <double>20.000000000000000</double>
         </property>
         <property name="maximum">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vmax</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="voltageSpinBox" name="vMinBox">
         <property name="minimum">
          <double>-20.000000000000000</double>
         </property>
         <property name="maximum">
          <double>-20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Vmin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="voltageSpinBox" name="timeWindowBox">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>0.000001000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Time Window</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>voltageSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">voltagespinbox.h</header>
   <slots>
    <slot>setMin(double)</slot>
    <slot>setMax(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>scopeRangeEnterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>scopeRangeEnterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vMaxBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>scopeRangeEnterDialog</receiver>
   <slot>toUpdateYTop(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vMinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>scopeRangeEnterDialog</receiver>
   <slot>toUpdateYBot(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vMaxBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>vMinBox</receiver>
   <slot>setMax(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vMinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>vMaxBox</receiver>
   <slot>setMin(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeWindowBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>scopeRangeEnterDialog</receiver>
   <slot>toUpdateWindow(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toUpdateYTop(double)</slot>
  <slot>toUpdateYBot(double)</slot>
  <slot>toUpdateWindow(double)</slot>
 </slots>
</ui>
